src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1069,23 +1083,23 @@
   </data>
   <data name="FileExplorerPreview_ToggleSwitch_Preview_MD.Header" xml:space="preserve">
     <value>Markdown</value>
-    <comment>File type, do not translate</comment>
+    <comment>{Locked}</comment>
   </data>
   <data name="FileExplorerPreview_ToggleSwitch_Preview_MD.Description" xml:space="preserve">
     <value>.md, .markdown, .mdown, .mkdn, .mkd, .mdwn, .mdtxt, .mdtext</value>
     <comment>{Locked}</comment>
   </data>
   <data name="FileExplorerPreview_ToggleSwitch_Preview_Monaco.Header" xml:space="preserve">
     <value>Source code files (Monaco)</value>
-    <comment>File type, do not translate</comment>
+    <comment>{Locked}</comment>
```

Comment:
IMO this shouldn't be locked.  I actually don't think we need (Monaco) as that doesn't provide any deeper context